base/compiler/ssair/verify.jl

```diff
@@ -254,6 +254,14 @@ function verify_ir(ir::IRCode, print::Bool=true,
         bb_unreachable(domtree, bb) && continue
         stmt = ir[SSAValue(idx)][:stmt]
         stmt === nothing && continue
+        foreachssa(stmt) do ssa
+            x = ir.stmts[ssa.id]
+            if x isa GotoIfNot || x isa GotoNode || x isa ReturnNode ||
+               (x isa Expr && (x.head == :leave || x.head == :gc_preserve_end))
+                @verify_error "Non value statement $(ssa.id) (defined as $(ir.stmts[ssa.id])) used as a value in statement $idx: $stmt"
+                error()
```

Comment:
```suggestion
                error("Verification error")
```

Reply:
I'm going to go for
```suggestion
                error("")
```
just to be consistent with the rest of the file